app/models/concerns/single_sign_on_concern.rb

```diff
@@ -67,10 +69,9 @@ def set_data_from_sso_mapping(sso_mapping, data)
           self.statistic_profile ||= StatisticProfile.new
           self.statistic_profile.birthday = data
         else
-          profile[sso_mapping[8..-1].to_sym] = data unless data.nil?
+          profile[sso_mapping[8..-1].to_sym] = data
         end
       end
-      return if data.nil? || data.blank? || mapped_from_sso&.include?(sso_mapping)
```

Comment:
According with my previous comment, I guess `return if mapped_from_sso&.include?(sso_mapping)` must be kept here